eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Proposals.hs

```diff
@@ -407,16 +352,14 @@ mkProposals pgais omap = do
       initialProposals
       omap
   unless (omap == omap') $ fail "mkProposals: OMap is malformed"
-  unless (proposalsAreConsistent ps) $ fail "mkProposals: Proposals structure is inconsistent"
```

Comment:
Do we want to check the invariant here by using `toPForestEither`? Similar to the checks after addition or deletion of a proposal?

Reply:
Those checks are done on the addition of each GovActionState, which happens few lines above

Reply:
Oh yes, got it!